lib/xlibc/include/time.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _XLIBC_TIME_H
+#define _XLIBC_TIME_H
+
+#include <_restrict.h>
+#include <_unimplemented.h>
```

Comment:
This might be displayed in wrong-order on GitHub, but this include file does not exist in this commit.

Reply:
GitHub seems to order the commits by time, which is all mixed up since I resorted the commits. When I look at my branch, the commits are in the correct order.